src/me/corriekay/pokegoutil/windows/PokemonTab.java

```diff
@@ -73,12 +74,13 @@ public PokemonTab(PokemonGo go) {
         setLayout(new BorderLayout());
         this.go = go;
         JPanel topPanel = new JPanel(new GridBagLayout());
-        JButton refreshPkmn, renameSelected, transferSelected, evolveSelected, powerUpSelected;
+        JButton refreshPkmn, renameSelected, transferSelected, evolveSelected, powerUpSelected, toggleFavoriteSelected;
         refreshPkmn = new JButton("Refresh List");
         renameSelected = new JButton("Rename");
         transferSelected = new JButton("Transfer");
         evolveSelected = new JButton("Evolve");
         powerUpSelected = new JButton("Power Up");
+        toggleFavoriteSelected = new JButton("Toggle Favorite Selected");
```

Comment:
Can you remove the "Selected"? Just keep button names as short as possible for small resolutions and such.